static/app/components/eventOrGroupHeader.tsx

```diff
@@ -60,8 +60,7 @@ function EventOrGroupHeader({
       <Fragment>
         {!hideLevel && level && <GroupLevel level={level} />}
         {!hideIcons &&
-          status === 'ignored' &&
-          !organization.features.includes('escalating-issues') && (
+          status === 'ignored' && (
```

Comment:
if my boolean logic is right, we should remove this block since `!organization.features.includes('escalating-issues')` would've been returning `False`?